ts/features/payments/checkout/store/selectors/paymentMethods.ts

```diff
@@ -121,21 +120,24 @@ export const notHasValidPaymentMethodsSelector = createSelector(
   walletPaymentAllMethodsSelector,
   walletPaymentEnabledUserWalletsSelector,
   walletPaymentDetailsSelector,
-  (allMethodsPot, userWalletsPot, paymentDetailsPot) => {
-    const allMethods = pipe(
-      allMethodsPot,
-      pot.toOption,
-      O.getOrElse(() => [] as PaymentMethodsResponse["paymentMethods"])
-    );
+  (_allMethodsPot, userWalletsPot, paymentDetailsPot) => {
+    // TODO: Uncomment the following lines when the "payment as a guest" feature will be implemented
```

Comment:
Create an issue and add the id to the comment

Reply:
Addressed into 3ef8ac774a2f696389c8098db839be3bf3d6f6d7